config/packages/sulu_security.yaml

```diff
@@ -0,0 +1,9 @@
+sulu_security:
+    password_policy:
+        enabled: false
```

Comment:
havent checked the `sulu/sulu` code yet, but this feels a bit unintuitive. do we need it for bc reasons? will the default policy still apply if this is set to false? 🤔

Reply:
we could go if no policy is configured no policy is used. I think there is no need to ship enabled flag then. And in the skeleton we just configure something valid for new projects.

Reply:
yeah i think that would be easier to understand 👍

Reply:
the default policy will not apply if it is to false - then the old behaviour will be kept (no policy at all). And yes its for BC-Reasons - if you activate it by setting the pattern we could not provide a default pattern and translation - then the developer has to set both translation (key and translate it by himself) and the pattern because that has to match.

By adding the enabled flag we can provide a proper default pattern and a coresponding translation for the information message

Reply:
the idea by alex and me was to set the default pattern in the skeleton, that would be more explicit 🙂 
it would backwards compatible because the project does not need to copy the changes of the skeleton

Reply:
problem with that is that we cannot provide a message in sulu and the pattern here - that would never match - for 3.0 we can then remove the `enabled` flag

Reply:
You just need to handle an empty `password_policy` like you currently handling `enabled` flag false. And default is empty.